src/main/java/com/patchworkmc/jar/ForgeModJar.java

```diff
@@ -1,53 +1,23 @@
 package com.patchworkmc.jar;
 
 import java.nio.file.Path;
-import java.util.HashSet;
-import java.util.List;
 
-import io.github.fukkitmc.gloom.definitions.ClassDefinition;
-import io.github.fukkitmc.gloom.definitions.GloomDefinitions;
-
-import com.patchworkmc.manifest.mod.ModManifest;
-import com.patchworkmc.manifest.mod.ModManifestEntry;
+import net.patchworkmc.manifest.accesstransformer.v2.ForgeAccessTransformer;
+import net.patchworkmc.manifest.mod.ModManifest;
 
 public class ForgeModJar {
-	private Path jarPath;
-	private ModManifest manifest;
-	private GloomDefinitions gloomDefs;
-	private HashSet<String> depedencies;
+	private final Path jarPath;
+	private final ModManifest manifest;
+	private final ForgeAccessTransformer at;
 
-	public ForgeModJar(Path jarPath, ModManifest manifest, GloomDefinitions gloomDefs) {
-		this.jarPath = jarPath;
-		this.manifest = manifest;
-		this.gloomDefs = gloomDefs;
-		this.depedencies = new HashSet<>();
-		// Store all of our dependencies into one Set
-		this.manifest.getDependencyMapping().forEach((modId, list) ->
-					list.forEach((modManifestDependency ->
-							this.depedencies.add(modManifestDependency.getModId()))));
+	public ForgeModJar(Path jarPath, ModManifest manifest) {
+		this(jarPath, manifest, null);
 	}
 
-	public void addDependencyJars(List<ForgeModJar> modJars) {
```

Comment:
This is okay for now but at some point in the future we'll need to merge ATs from dependency JARs.

Reply:
Why would that be needed? Deps would have their ATs applied to Minecraft, and the mod will be happy.

Reply:
Huh, that is true. Then this is only relevant for decompiling then using those transformed mods in development, where dependency AWs are not applied when compiling. But that's not a big deal for now.